utils/demarchesSimplifiees.js

```diff
@@ -50,10 +50,10 @@ function getNextCursor(apiResponse) {
 
 /**
  * Converts the first character of first name and last name to upper case and the remaining to lower case.
- * @param {*} demandeur 
+ * @param {*} demandeur
  */
 function getName(demandeur) {
-  return _.capitalize(demandeur.prenom) + ' ' + _.capitalize(demandeur.nom);
+  return _.capitalize(demandeur.nom.trim()) + ' ' + _.capitalize(demandeur.prenom.trim());
```

Comment:
y'a une raison spéciale de faire un `trim()` ? 

j'ai refactoré cette fonction pour m'adapter au nouveau schéma : 
* firstNames
* lastName

j'ai enlévé le `capitalize` car je suis pas certain que tous les noms et prénoms comportent des majuscules, je sais que certains "leclercq" (mon nom de famille car ce n'est pas "polomarcus" dédicace à @LucasCharrier :grin: ) écrivent leur nom de famille "LeClercq" par exemple

Reply:
Le `trim()` évite qu'on ai des espaces avant ou après le texte. Ça permet d'avoir un texte unifié.

Reply:
j'ai rajouté le trim parce que la capitalisation ne marchait pas à tous les coups, j'en deduis que certains commencaient par des espaces. En le rajoutant ca capitalisait bien tout.

Du coup on peut zapper la fonction _.capitalize et convertir la premiere lettre en majuscule "a la main". Les noms non-capitalisés font merder l'ordre alphabétique (ils arrivent en premier) donc ca vaut vraiment le coup de régler ca je pense.

Reply:
d'ailleurs, enlever ce capitalize, ca permettra de zapper lodash entier, puisqu'on l'avait importé que pour ca :) (avec les nouvelles fonctions qui arrivent dans JS, on doit pouvoir se debrouiller sans lodash...)

Reply:
dans la PR #20 on stocke les noms et prenoms en brute, on géra ça à l'affichage si ça vous va ?

 ```
  const lastName = dossier.demandeur.nom;
  const firstNames = dossier.demandeur.prenom;
```

https://github.com/betagouv/sante-psy/blob/feat/import-data-from-DS-to-PG/utils/demarchesSimplifiees.js#L102-L103

autre proposition, mettre tout en minuscule dans la DB ?